src/test/java/api/test/LoginUserViaApi.java

```diff
@@ -0,0 +1,129 @@
+package api.test;
+
+import api.AuthorizationAPI;
+import api.UserAPI;
+import com.github.javafaker.Faker;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LoginUserViaApi {
+
+    AuthorizationAPI authorizationAPI;
+    UserAPI userAPI;
+    Faker faker = new Faker();
+
+    String expectedUsername = "John";
+    String expectedPassword = "123NBK&tr";
+
+
+    @Test
+    public void loginUserWithValidDataTest() {
+
+        //login user
+        authorizationAPI = new AuthorizationAPI();
+        authorizationAPI.login(expectedUsername, 200, expectedPassword);
+
+        //check with valid username and valid password
+        boolean authorization = authorizationAPI.isAuthorized(expectedUsername, expectedPassword, 200);
+        Assert.assertTrue(authorization, "User is not authorized, status: " + authorization);
+    }
+
+
+    @Test
+    public void loginUserWithInvalidPasswordTest() {
+
+        //login user
+        authorizationAPI = new AuthorizationAPI();
+        authorizationAPI.login(expectedUsername, 200, expectedPassword);
```

Comment:
Invalid data here should be data like in isAuthorized
And the better if you add this data in variable inside this method

Reply:
fixed